spec/lib/terraforming/resource/datadog_monitor_spec.rb

```diff
@@ -0,0 +1,197 @@
+require "spec_helper"
+
+module Terraforming
+  module Resource
+    describe DatadogMonitor do
+      let(:client) { instance_double(Dogapi::Client).as_null_object }
+
+      let(:monitors) do
+        [ "200",
+          [
+            {
+              "tags"=>[],
+              "deleted"=>nil,
+              "query"=>"avg(last_1h):avg:aws.xxxx > 15000",
+              "message"=>"@slack-team-xxxx",
+              "matching_downtimes"=>[],
+              "id"=>521075,
+              "multi"=>false,
+              "name"=>"terraform alarm 1",
+              "created"=>"2016-03-18T14:19:19.948460+00:00",
+              "created_at"=>1458310759000,
+              "creator"=> {
+                "id"=>177888,
+                "handle"=>"j@example.com",
+                "name"=>"J Doe",
+                "email"=>"j@example.com"
```

Comment:
__[rubocop]__
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->